projects/dxc-ngx-cdk/src/lib/dxc-page-errors/dxc-page-errors.component.ts

```diff
@@ -5,7 +5,8 @@ import { AbstractControlDirective, AbstractControl } from '@angular/forms';
   selector: 'dxc-show-errors',
   template: `
    <ul *ngIf="shouldShowErrors()" class="position-absolute">
-     <li style="color: red" *ngFor="let error of listOfErrors()">{{error}}</li>
+     <li style="color: red" *ngFor="let error of listOfErrors()">
+     <div role="alert" tabindex="0">{{error}}</div></li>
```

Comment:
Why u need tabindex ? even this element doesn't have event

Reply:
without tabindex the alert message not getting focused

Reply:
check if that control have aria-required attribute